### PR TITLE
added urls path

### PR DESCRIPTION
**Pull request by carys23**

### CODE REVIEW

File: cage/cryptids/views.py [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
-    model = City
-    template_name = 'Search_results.html'
+
+
+def detail(render, animals_id):
+    animals = get_objects_or_404(Animals, pk=animals_id)
+    return render(request, 'animals/detail.html', {'animals': animals})
+
+def index(request):
+    context ={"animlas": Animlas.objects.all()}
```
Comment (tamjankowska) [inline cage/cryptids/views.py:20]: Little typo of `animals` here.

File: cage/cryptids/views.py [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
 
-class HomepageView(TemplateView):
-    template_name = 'home.html'
 
-class SearchResutlsView(ListView):
-    model = City
-    template_name = 'Search_results.html'
+
+
+def detail(render, animals_id):
```
Comment (tamjankowska) [inline cage/cryptids/views.py:15]: It needs to be `request` as an argument rather than `render`.

File: cage/cryptids/views.py [hunk 3]
```diff
@@ -1,12 +1,23 @@
 from django.shortcuts import render
+from django.http import HttpResponse
+from .models import Animals, Location
+from django.template import loader
+from django.http import Http404
+from django.shortcuts import get_object_or_404, render
 from django.views.generic import TemplateView, ListView
-from .models import City
+from django.http import HttpResponse, HttpResponseRedirect
+from django.shortcuts import get_object_or_404, render
```
Comment (tamjankowska) [inline cage/cryptids/views.py:9]: Looks like you have some duplicated imports?

File: cage/cryptids/urls.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+app_name ='crydtids'
```
Comment (tamjankowska) [inline cage/cryptids/urls.py:3]: Little typo of `cryptids` here.

File: cage/cryptids/templates/index.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% load static %}
+
+
+{% block title %}Cryptids{% endblock %}
+{% block content %}
+
+
```
Comment (tamjankowska) [inline cage/cryptids/templates/index.html:8]: Bit of blank space that could go here :)

File: cage/cryptids/templates/index.html [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+
+<ul>
+        {% for animals in animalss %}
```
Comment (tamjankowska) [inline cage/cryptids/templates/index.html:12]: I would suggest updating this for loop like we mentioned last night.

**Review by tamjankowska — CHANGES_REQUESTED** [review]
A few things to consider here :)

**Comment by carys23** [comment]
re do

**Pull request closed without merging**